t/lib/t/MusicBrainz/Server/Controller/Admin/DeleteEditor.pm

```diff
@@ -34,6 +43,11 @@ test 'Delete account as a regular user' => sub {
     html_ok($mech->content);
     $mech->content_contains('You need to be logged in to view this page');
 
+    $second_session->get('/');
+    is($second_session->status, 500, 'restoring deleted user fails');
```

Comment:
If this situation occurs (someone is active from two browsers more or less at the same time, deletes their account from one of them, then continues browsing the website using the other browser before the session has expired), Catalyst throws an exception, resulting in an ISE page. I think this will be so rare that it’s not worth to try and catch it and deal with it separately.